Lesson 4 - DOM/code/homework/app.js

```diff
@@ -99,6 +101,8 @@ window.addEventListener('DOMContentLoaded', function () {
     // add event listeners observers for both `keydown` and `keyup` events
     // using 'keydown' and 'keyup' functions provided above
 
+    keyDown.addEventListener(keyDown, false);
```

Comment:
Incorrect object. Common mistake. try `document`, instead of `keydown`.

![image](https://cloud.githubusercontent.com/assets/4668423/14258874/b9f08404-faa4-11e5-8f59-939b04bca8c0.png)